controllers/osccluster_net_controller_unit_test.go

```diff
@@ -170,11 +170,11 @@ func TestCheckNetFormatParameters(t *testing.T) {
 				Network: infrastructurev1beta1.OscNetwork{
 					Net: infrastructurev1beta1.OscNet{
 						Name:    "test-net",
-						IpRange: "10.0.0.256/16",
+						IpRange: "10.0.0.256/8",
```

Comment:
Please change to use the new default values.

Reply:
It's for a use case : "check Bad Ip Range IP Net"

Reply:
Ok